### ISSUE TITLE
MacOS installation is outdated

### ISSUE THREAD

**Issue by roablep** [deanmalmgren/textract#437]
**Describe the bug**
The mac install instructions are outdated. `cask` is no longer a `brew` command. I think you just do `brew install xquartz` or `brew install --cask xquartz ` instead of `brew cask install xquartz`.

**To Reproduce**
```
(.venv) (base) peter@Peters-MBP14 % brew config
HOMEBREW_VERSION: 3.5.2
ORIGIN: https://github.com/Homebrew/brew
HEAD: fba051f98f609634ae4392d0053ddfc627f8cdb3
Last commit: 3 months ago

(.venv) (base) peter@Peters-MBP14 % brew cask install xquartz
Error: `brew cask` is no longer a `brew` command. Use `brew <command> --cask` instead.
```
**Desktop (please complete the following information):**
 - OS: Mac 12.5.1
 - Textract textract-1.6.5-py3-none-any.whl
 - Python 3.9
 - Virtual environment yes

**Comment by roablep** [comment]
Although it looks like they're correct in [github](https://github.com/deanmalmgren/textract/blob/master/docs/installation.rst)

**Comment by KyleKing** [comment]
Yeah, the problem is Read the Docs not being re-run. I'll investigate and resolve this

**Comment by KyleKing** [comment]
@deanmalmgren, when you have time, could you please add me to the Read the Docs project so that I can continue to debug the build issues? I created a RTD account with the same name as GitHub: https://app.readthedocs.org/profiles/KyleKing

Last build was over four years ago: https://app.readthedocs.org/projects/textract/builds/?utm_source=textract&utm_content=flyout

**Comment by deanmalmgren** [comment]
@KyleKing I just tried to login to RTD for the first time in years and I'm afraid that I ran into a stumbling block; the email to confirm my identity was almost certainly sent to an old work email address. I've emailed their support team to regain access to my account.

I'll be sure to add you as a maintainer on RTD when I get access, but it'll likely take a few days.

**Comment by deanmalmgren** [comment]
@KyleKing I was able to gain access to RTD this afternoon and added you as a maintainer. Looks like you need to approve and then you'll be able to diagnose the issue.

**Comment by KyleKing** [comment]
Thanks! 🎉 I won't have time today, but I accepted the invite and will follow up this week

**Comment by KyleKing** [comment]
Seems to be updated now!

<img width="1133" height="798" alt="Image" src="https://github.com/user-attachments/assets/a45a73f9-3d5b-4930-8ce5-83ec91d0394b" />
<img width="1143" height="768" alt="Image" src="https://github.com/user-attachments/assets/9de1922c-2cd0-4f74-9445-038f37eb729c" />
